### PR TITLE
migrate  MyNativeViewManager to kotlin

### PR DESCRIPTION
**Pull request by tarunrajput**
## Summary:

Migrate MyNativeViewManager to kotlin as part of ☂️ #38825 

## Changelog:

[Internal][Changed]: Migrate MyNativeViewManager to kotlin

## Test Plan:

Verify RN Tester runs with `yarn android`

### CODE REVIEW

**Comment by tarunrajput** [comment]
@mdvacca, do you mind checking what's Facebook Internal Linter warning about? 🙏

File: packages/rn-tester/android/app/src/main/java/com/facebook/react/uiapp/component/MyNativeViewManager.kt [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+internal class MyNativeViewManager :
+    SimpleViewManager<MyNativeView>(), RNTMyNativeViewManagerInterface<MyNativeView> {
+
+    companion object {
+        const val REACT_CLASS = "RNTMyNativeView"
+    }
+
+    private val delegate: ViewManagerDelegate<MyNativeView> = RNTMyNativeViewManagerDelegate(this)
+
+    override protected fun getDelegate(): ViewManagerDelegate<MyNativeView>? = delegate
```
Comment (cortinico) [inline packages/rn-tester/android/app/src/main/java/com/facebook/react/uiapp/component/MyNativeViewManager.kt:33]:
```suggestion
    override fun getDelegate(): ViewManagerDelegate<MyNativeView> = delegate
```

File: packages/rn-tester/android/app/src/main/java/com/facebook/react/uiapp/component/MyNativeViewManager.kt [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    }
+
+    @ReactProp(name = "values")
+    override fun setValues(view: MyNativeView, value: ReadableArray?) {
+        val values = ArrayList<Int>()
+        value?.let {
+            for (i in 0 until value.size()) {
+                values.add(value.getInt(i))
+            }
+        }
```
Comment (cortinico) [inline packages/rn-tester/android/app/src/main/java/com/facebook/react/uiapp/component/MyNativeViewManager.kt:60]:
```suggestion
        val values = mutableListOf<Int>()
        value?.toArrayList()?.forEach {
            values.add(it as Int)
        }
```

File: packages/rn-tester/android/app/src/main/java/com/facebook/react/uiapp/component/MyNativeViewManager.kt [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                            "bubbled",
+                            "onIntArrayChanged",
+                            "captured",
+                            "onIntArrayChangedCapture"
+                        )
+                    )
+                )
+                .build()
+        return eventTypeConstants
+    }
```
Comment (cortinico) [inline packages/rn-tester/android/app/src/main/java/com/facebook/react/uiapp/component/MyNativeViewManager.kt:81]:
```suggestion
    override fun getExportedCustomBubblingEventTypeConstants(): Map<String, Any> =
        MapBuilder.builder<String, Any>()
            .put(
                "onIntArrayChanged",
                MapBuilder.of<String, Any>(
                    "phasedRegistrationNames",
                    MapBuilder.of(
                        "bubbled",
                        "onIntArrayChanged",
                        "captured",
                        "onIntArrayChangedCapture"
                    )
                )
            )
            .build()
```

File: packages/rn-tester/android/app/src/main/java/com/facebook/react/uiapp/component/MyNativeViewManager.kt [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+        const val REACT_CLASS = "RNTMyNativeView"
+    }
+
+    private val delegate: ViewManagerDelegate<MyNativeView> = RNTMyNativeViewManagerDelegate(this)
+
+    override protected fun getDelegate(): ViewManagerDelegate<MyNativeView>? = delegate
+
+    override fun getName(): String = REACT_CLASS
+
+    override protected fun createViewInstance(reactContext: ThemedReactContext): MyNativeView =
```
Comment (cortinico) [inline packages/rn-tester/android/app/src/main/java/com/facebook/react/uiapp/component/MyNativeViewManager.kt:37]:
```suggestion
    override fun createViewInstance(reactContext: ThemedReactContext): MyNativeView =
```

**Review by cortinico — COMMENTED** [review]
Thanks for sending this over @tarunrajput  👍 left some comments that needs to be addressed before we can merge this one

**Review by mdvacca — APPROVED** [review]
lgtm

**Review by cortinico — APPROVED** [review]

**Pull request closed without merging**